src/patternfly/components/Drawer/drawer.scss

```diff
@@ -12,24 +12,26 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   --pf-c-drawer__content--ZIndex: var(--pf-global--ZIndex--xs);
 
   // Panel
-  --pf-c-drawer__panel--FlexBasis: 100%;
-  --pf-c-drawer__panel--md--FlexBasis: 50%;
   --pf-c-drawer__panel--MinWidth: 50%; // change to __panel--md--MinWidth at breaking change
   --pf-c-drawer__panel--MaxHeight: auto;
-  --pf-c-drawer--m-panel-bottom__panel--md--MinHeight: 50%;
-  --pf-c-drawer__panel--xl--MinWidth: #{pf-size-prem(450px)};
-  --pf-c-drawer__panel--xl--FlexBasis: #{pf-size-prem(450px)};
-  --pf-c-drawer--m-panel-bottom__panel--xl--MinHeight: #{pf-size-prem(300px)};
-  --pf-c-drawer--m-panel-bottom__panel--xl--FlexBasis: #{pf-size-prem(300px)};
   --pf-c-drawer__panel--ZIndex: var(--pf-global--ZIndex--sm);
   --pf-c-drawer__panel--BackgroundColor: var(--pf-global--BackgroundColor--100);
   --pf-c-drawer__panel--TransitionDuration: var(--pf-global--TransitionDuration);
   --pf-c-drawer__panel--TransitionProperty: margin, transform, box-shadow, flex-basis;
-  --pf-c-drawer__panel--m-resizable--FlexBasis: initial;
+  --pf-c-drawer__panel--FlexBasis: 100%;
+  --pf-c-drawer__panel--md--FlexBasis--min: 0%;
+  --pf-c-drawer__panel--md--FlexBasis: 50%;
+  --pf-c-drawer__panel--md--FlexBasis--max: 100%;
+  --pf-c-drawer__panel--xl--MinWidth: #{pf-size-prem(450px)};
+  --pf-c-drawer__panel--xl--FlexBasis: #{pf-size-prem(450px)};
+  --pf-c-drawer--m-panel-bottom__panel--md--MinHeight: 50%;
+  --pf-c-drawer--m-panel-bottom__panel--xl--MinHeight: #{pf-size-prem(300px)};
+  --pf-c-drawer--m-panel-bottom__panel--xl--FlexBasis: #{pf-size-prem(300px)};
+  --pf-c-drawer__panel--m-resizable--FlexBasis: initial; // falls back by default
```

Comment:
This variable is not used in any class, do we still need to keep it here?

Reply:
Nope, good find! Removed.